formidable/models.py

```diff
@@ -53,10 +53,14 @@ def from_json(definition_schema, **kwargs):
         from formidable.serializers import FormidableSerializer
 
         serializer = FormidableSerializer(data=definition_schema)
-        if serializer.is_valid():
-            return serializer.save(**kwargs)
+        if not serializer.is_valid():
+            raise ValidationError(serializer.errors)
 
-        raise ValidationError(serializer.errors)
+        return serializer.save(**kwargs)
+
+    def to_json(self):
+        from formidable.serializers import FormidableSerializer
```

Comment:
I guess that flake8 won't be very happy with this. Why importing it here? wouldn't it work on the top of the file?